Signal/src/environment/Environment.m

```diff
@@ -244,6 +244,8 @@ - (NotificationsManager *)notificationsManager
 
 + (PropertyListPreferences *)preferences
 {
+    OWSAssert([Environment getCurrent] != nil);
+    OWSAssert([Environment getCurrent].preferences != nil);
```

Comment:
Not opposed to some redundancy, but I notice `getCurrent` is already asserting that it's not nil.